src/Maatwebsite/Excel/Parsers/ExcelParser.php

```diff
@@ -296,7 +296,7 @@ protected function getSluggedIndex($value, $ascii = false)
         $value = preg_replace('![' . preg_quote($flip) . ']+!u', $separator, $value);
 
         // Remove all characters that are not the separator, letters, numbers, or whitespace.
-        $value = preg_replace('![^' . preg_quote($separator) . '\pL\pN\s]+!u', '', mb_strtolower($value));
+        $value = preg_replace('![^' . preg_quote(Config::get('excel.import.custom_seperators', true)) . '\pL\pN\s]+!u', '', mb_strtolower($value));
```

Comment:
Pass `$separator` as 2nd param of config. And use the config helper instead of facade.